ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/remote/MLSdkAsyncHttpResponseHandler.java

```diff
@@ -80,6 +79,7 @@ public void onHeaders(SdkHttpResponse response) {
         SdkHttpFullResponse sdkResponse = (SdkHttpFullResponse) response;
         log.debug("received response headers: " + sdkResponse.headers());
         this.statusCode = sdkResponse.statusCode();
+        this.errorsInHeader = sdkResponse.headers().get("x-amzn-ErrorType");
```

Comment:
why the error is amazon specific?

Reply:
It's the SageMaker error, and has fixed key.